### PR TITLE
Created Leaderboard for User Ediorials Screen

### PR DESCRIPTION
**Pull request by imraghav20**

### CODE REVIEW

File: stopstalk_app/lib/classes/user_editorial_class.dart [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class UserEditorial {
+  final String editorial_count;
```
Comment (Jashpatel1) [inline stopstalk_app/lib/classes/user_editorial_class.dart]: Refer to this for naming non-const identifiers: [https://dart-lang.github.io/linter/lints/non_constant_identifier_names.html](https://dart-lang.github.io/linter/lints/non_constant_identifier_names.html)

File: stopstalk_app/lib/classes/user_editorial_class.dart [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  final String votes_count;
```
Comment (Jashpatel1) [inline stopstalk_app/lib/classes/user_editorial_class.dart]: Same as stated above

File: stopstalk_app/lib/classes/user_editorial_class.dart (hunk 1 above)
Comment (imraghav20) [reply stopstalk_app/lib/classes/user_editorial_class.dart]: Renamed the variable to "editorialCount".

File: stopstalk_app/lib/classes/user_editorial_class.dart (hunk 2 above)
Comment (imraghav20) [reply stopstalk_app/lib/classes/user_editorial_class.dart]: renamed the variable to "votesCount".

**Pull request merged**